doc/requirements.rst

```diff
@@ -47,6 +47,8 @@ Motor and PyMongo
 +-------------------+-----------------+
 | 2.5               | 3.12+           |
 +-------------------+-----------------+
+| 3.0               | 4.0+            |
++-------------------+-----------------+
 
 Motor and MongoDB
```

Comment:
Let's add 3.0 to this "Motor and MongoDB" table too.

Reply:
Done